doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -16,7 +20,7 @@ enable more features, such as distributed memory execution.
 0) See :ref:`prereqs.rst <readme-prereqs>` for more information about system
    tools and packages you may need to have installed to build and run Chapel.
 
-1) If you don't already have Chapel 1.15, see
+1) If you don't already have Chapel 1.16, see
```

Comment:
Although, that's just one solution. I'm open to other ideas. My reasoning for the previous approach is that I'm sure users will find their way onto the master-version of the quickstart instructions, and will be thrown off when the 1.16 tarball does not exist...